js/stateless.js/src/idls/psp_compressed_pda.ts

```diff
@@ -1,1501 +1,1501 @@
 export type PspCompressedPda = {
-  "version": "0.3.0",
-  "name": "psp_compressed_pda",
-  "instructions": [
-    {
-      "name": "executeCompressedTransaction",
-      "docs": [
-        "This function can be used to transfer sol and execute any other compressed transaction.",
```

Comment:
your pr doesn't change the programs, it should not modify the idl files.
Please revert the idl files changes.